src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+} */
+
+.search__bar {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+}
+
+.search__icon {
+  background-image: url(images/Search.svg);
+  width: 19px;
+  height: 19px;
+  flex-shrink: 0;
+  display: block;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
```

Comment:
Combine it with [`background`](https://developer.mozilla.org/en-US/docs/Web/CSS/background) shorthand property